src/io/kqueue.rs

```diff
@@ -198,7 +202,13 @@ pub enum EventFilter {
     },
 
     /// A timer filter.
-    Timer(Option<Duration>),
+    Timer {
+        /// The identifier for this event.
+        ident: intptr_t,
```

Comment:
Does this ever hold an actual pointer value? If so, rustix's convention is to use an actual raw pointer type rather than `intptr_t`, to better follow strict provenance.

Reply:
No, it's an arbitrary ID number; I've just copied `intptr_t` from another variant here. On FreeBSD for example kqueue idents are `__uintptr_t`, but I guess signed was chosen due to what other platforms have